src/cookbook/ScottPlot.Cookbook/Recipes/Palette.cs

```diff
@@ -273,16 +273,18 @@ public void ExecuteRecipe(Plot plt)
         }
     }
 
-    public class PaletteColorBlindness : IRecipe
+    public class PaletteColorblindFriendly : IRecipe
     {
         public string Category => "Palette";
         public string ID => "palette_ColorBlindness";
         public string Title => "ColorBlindness";
```

Comment:
You missed the title

Reply:
Great catch! Thanks for looking over my work so carefully 😄